crates/fda/src/cli.rs

```diff
@@ -313,6 +313,35 @@ pub enum PipelineAction {
         #[arg(long, short = 's', default_value_t = false)]
         start: bool,
     },
+    /// Execute an ad-hoc query in a pipeline and return the result.
```

Comment:
what do you mean "in a pipeline"?
What does this do?
Is this documented somewhere?

Reply:
updated comment, this is printed as part of --help